crates/hyperdrive-math/src/lib.rs

```diff
@@ -144,6 +144,59 @@ impl State {
         }
     }
 
+    /// Calculates the pool reserve levels to achieve a target interest rate.
+    /// This calculation does not take Hyperdrive's solvency constraints or exposure
+    /// into account and shouldn't be used directly.
+    ///
+    /// The price for a given fixed-rate is given by $p = 1 / (r \cdot t + 1)$, where
+    /// $r$ is the fixed-rate and $t$ is the annualized position duration. The
+    /// price for a given pool reserves is given by $p = \frac{\mu z}{y}^t_{s}$,
+    /// where $\mu$ is the initial share price and $t_{s}$ is the time stretch
+    /// constant. By setting these equal we can solve for the pool reserve levels
+    /// as a function of a target rate.
+    ///
+    /// For some target rate, $r_t$, the pool share reserves, $z_t$, must be:
+    ///
+    /// $$
+    /// z_t = \frac{1}{\mu} \left(
+    ///   \frac{k}{\frac{c}{\mu} + \left(
+    ///     (r_t \cdot t + 1)^{\frac{1}{t_{s}}}
+    ///   \right)^{1 - t_{s}}}
+    /// \right)^{\tfrac{1}{1 - t_{s}}}
```

Comment:
```suggestion
    /// \right)^{\frac{1}{1 - t_{s}}}
```

Reply:
fwiw this is strictly a style preference; I generally like to use `tfrac` everywhere. I make exceptions for small fractions (usually `c/µ`) because it makes the fraction bar vertical instead of slanted.

Reply:
Ah! I didn't realize tfrac was valid. I thought it threw an error in overleaf.